crates/driver/rustfmt.toml

```diff
@@ -0,0 +1,19 @@
+tab_spaces = 4
+edition = "2021"
+fn_single_line = false
+format_code_in_doc_comments = true
+format_strings = true
+imports_layout = "HorizontalVertical"
+imports_granularity = "One"
+normalize_doc_attributes = true
+reorder_imports = true
+reorder_impl_items = true
+group_imports = "StdExternalCrate"
+use_try_shorthand = true
+wrap_comments = true
+max_width = 100
+overflow_delimited_expr = true
+remove_nested_parens = true
+reorder_modules = true
+unstable_features = true
+use_field_init_shorthand = true
```

Comment:
that's already a clippy warning but I guesse we might as well let rustfmt do it automatically